nx-X11/config/cf/xfree86.cf

```diff
@@ -1021,14 +982,6 @@ XCOMM $Xorg: xfree86.cf,v 1.4 2000/08/17 19:41:49 cpqbld Exp $
 				DevelDrivers vga \
 				XF86OSCardDrivers XF86ExtraCardDrivers
 # endif
-# if HasGlide3
```

Comment:
> Could probably also remove HasGlide3 from:
>
>   nx-X11/config/cf/xf86site.def
>   nx-X11/config/cf/linux.cf
>   nx-X11/config/cf/Imake.tmpl
>   nx-X11/config/cf/host.def

The Glide2/3 stuff will be removed via PR https://github.com/ArcticaProject/nx-libs/pull/42. So doing nothing here.